book/README.md

```diff
@@ -0,0 +1 @@
+# Introduction
```

Comment:
Should there be more content here? I have trouble imagining what this will look like in the rendered book.

Reply:
It's the readme.md

Reply:
If I understand correctly, this file is included in the `SUMMARY.md`, where it appears as a new, empty chapter with the headline `Introduction`, but without content. Is that correct? Or is this meant to include the main README.md from the root of the repo?